gulpfile.js

```diff
@@ -117,3 +118,9 @@ gulp.task('build-js', function() {
   .bundle()
   .pipe(callback('static/js/common.js'));
 });
+
+gulp.task('test', function() {
+  return mochify('./tests/unit/**.js')
```

Comment:
Agreed. I separated the election lookup logic into a resuable module (modules/election-lookup) and a page that imports and instantiates the module (pages/election-lookup), so the tests have moved to tests/unit/modules/election-lookup.